packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -116,6 +117,11 @@ E? _lastWhereOrNull<E>(Iterable<E> list, bool Function(E) test) {
 /// If the [tags] are passed, they declare user-defined tags that are implemented by
 /// the `test` package.
 ///
+/// The argument [experimentalLeakTesting] is experimental and is not recommended
+/// for use outside of Flutter Framework.
+/// [experimentalLeakTesting] is noop.
```

Comment:
nit: sentences should start grammatically correct with a capital letter: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar

Reply:
Resolved in https://github.com/flutter/flutter/pull/137655